components/google_calendar/sources/new-or-updated-event-instant/new-or-updated-event-instant.mjs

```diff
@@ -107,39 +99,39 @@ export default {
     async makeWatchRequest() {
       // Make watch request for this HTTP endpoint
       for (const calendarId of this.calendarIds) {
-        const config = {
-          calendarId,
-          requestBody: {
-            id: uuid(),
-            type: "web_hook",
-            address: this.http.endpoint,
-          },
-        };
-        const watchResp = await this.googleCalendar.watch(config);
-        const data = watchResp.data;
+        const watchResp =
+          await this.googleCalendar.watchEvents({
+            calendarId,
+            requestBody: {
+              id: uuid(),
+              type: "web_hook",
+              address: this.http.endpoint,
+            },
+          });
 
         // Initial full sync. Get next sync token
         const nextSyncToken = await this.googleCalendar.fullSync(calendarId);
 
         this.db.set(`${calendarId}.nextSyncToken`, nextSyncToken);
-        this.db.set(`${calendarId}.channelId`, data.id);
-        this.db.set(`${calendarId}.resourceId`, data.resourceId);
-        this.db.set(`${calendarId}.expiration`, data.expiration);
+        this.db.set(`${calendarId}.channelId`, watchResp.id);
+        this.db.set(`${calendarId}.resourceId`, watchResp.resourceId);
+        this.db.set(`${calendarId}.expiration`, watchResp.expiration);
       }
     },
     async stopWatchRequest() {
       for (const calendarId of this.calendarIds) {
         const id = this.db.get(`${calendarId}.channelId`);
```

Comment:
I found more `db` here that could be encapsulated in a separate function. If I missed some one, please go ahead and encapsulate it as well